### ISSUE TITLE
Some markdown css classes aren't inlined unless seen by parser

### ISSUE THREAD

**Issue by srtonz** [maizzle/framework#431]
When using [`markdown-it-attrs`](https://github.com/arve0/markdown-it-attrs) or comparable plugins to add html classes from markdown, maizzle does not seem to inline styles unless they've been seen in html before.

**Test setup**
```js
// config.production.js
module.exports = {
  build: {
    templates: {
      destination: {
        path: 'build_production',
      },
    },
  },
  inlineCSS: true,
  removeUnusedCSS: false,
  markdown: {
    plugins: [
      { plugin: require('markdown-it-attrs') }
    ]
  }
}
```

```html
<!-- src/templates/transactional.html -->
<extends src="src/layouts/master.html">
  <block name="template">
    <md>
      Hello World {.w-full}

      Hello World {.w-32}

      Hello World {.w-16}
    </md>

    <p class="w-16 hidden"></p>
  </block>
</extends>
```

Then run `maizzle build production`

**Expected result**
```html
...
<p class="w-full" style="width: 100%;">Hello World</p>
<p class="w-32" style="width: 32px;">Hello World</p>
<p class="w-16" style="width: 16px;">Hello World</p>
<p class="w-16 hidden" style="display: none; width: 16px;"></p>
...
```

**Actual result**
```html
...
<p class="w-full" style="width: 100%;">Hello World</p>
<p class="w-32">Hello World</p>
<p class="w-16" style="width: 16px;">Hello World</p>
<p class="w-16 hidden" style="display: none; width: 16px;"></p>
...
```

**Comments**
* The third paragraph has the `style` attribute applied because of the hidden paragraph at the end. Removing the hidden `p` will remove the style from the previous paragraph.
* The first paragraph's `w-full` is applied correctly, despite not being used in the template or the layout. The reason appears to be that `w-full` is used in the `promotional.html` template; deleting that template will remove the `style` attribute from the first paragraph.
* Maizzle Version: 3.0.0
* Node.js Version: 15.12.0

**Comment by cossssmin** [comment]
This is caused by the (old) Tailwind class extractor we're using for `postcss-purgecss`. We'll move away from that purging strategy soon (as part of adding JIT support), and just rely on Tailwind's `purge` option to do the job.

Thanks for catching this, we'll fix it soon!

**Comment by cossssmin** [comment]
This will be fixed in the next release, we'll be using Tailwind's own purging going forward.

**Comment by cossssmin** [comment]
Fixed in v3.4.0

https://github.com/maizzle/framework/releases/tag/v3.4.0
